TypeScript error in `users.ts` - `getSessionRecord` does not exist

When running `npm run build`, a TypeScript error occurs in `src/store/api/users.ts`, indicating that `getSessionRecord` does not exist in `UsersApi`.

### Error Message

```
src/store/api/users.ts:21:49 - error TS2551: Property 'getSessionRecord' does not exist on type 'UsersApi'. Did you mean 'setSessionRecord'?

21 export const getSecurity = async () => usersApi.getSessionRecord();
                                                   ~~~~~~~~~~~~~~~~

  src/api/client/api.ts:17668:12
    17668     public setSessionRecord(tenant: string, setSessionRecordRequest?: SetSessionRecordRequest, options?: AxiosRequestConfig) {
                     ~~~~~~~~~~~~~~~~
    'setSessionRecord' is declared here.
```

--- a/ui/src/store/api/users.ts
+++ b/ui/src/store/api/users.ts
@@ -18,7 +18,7 @@ export const postValidationAccount = async (data : IUser) => usersApi.getValidat
 
 export const putSecurity = async (data : IUserPutSecurity) => usersApi.setSessionRecord(data.id, { session_record: data.status });
 
-export const getSecurity = async () => usersApi.getSessionRecord();
+export const getSecurity = async () => usersApi.checkSessionRecord();
 
 export const postUpdatePassword = async (data : IUserUpdatePassword) => usersApi.updateRecoverPassword(data.id, {
   token: data.token,

